las_geoh5/__init__.py

```diff
@@ -25,3 +26,97 @@ def assets_path() -> Path:
         raise RuntimeError(f"Assets folder not found: {assets_folder}")
 
     return assets_folder
+
+
+# TODO: Propose change on lasio to fix possible version issue
+
+
+def configure_metadata_patterns(line, section_name):  # pylint: disable=too-many-locals
+    """Configure regular-expression patterns to parse section meta-data lines.
+
+    # OVERLOAD lasio.reader.configure_metadata_patterns
+
+    Arguments:
+        line (str): line from LAS header section
+        section_name (str): Name of the section the 'line' is from.
+
+    Returns:
+        An array of regular-expression strings (patterns).
+    """
+
+    # Default return value
+    patterns = []
+
+    # Default regular expressions for name, value and desc fields
+    name_re = r"\.?(?P<name>[^.]*)\."
+    value_re = r"(?P<value>.*):"
+    desc_re = r"(?P<descr>.*)"
+
+    # Default regular expression for unit field. Note that we
+    # attempt to match "1000 psi" as a special case which allows
+    # a single whitespace character, in contradiction to the LAS specification
+    # See GitHub issue #363 for details.
+    if "VERS" in line:
+        unit_re = r"(?P<unit>\D*)"
+    else:
+        unit_re = r"(?P<unit>([0-9]+\s)?[^\s]*)"
```

Comment:
Only change is here

Reply:
I'm not great with re's, so I'll just go to test.